Changing plank weight and volume to 2200g and 4400mL

Jovus opened the pull request:
<!--
### How to use
Leave the headings unless they don't apply to your PR, replace commented out text (surrounded with <!–– and ––>) with text describing your PR.
NOTE: Please grant permission for repository maintainers to edit your PR.
It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
-->

#### Summary
<!--
A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).

The format is (ignore the square brackets): ```SUMMARY: [Category] "[description]"```

The categories to choose from are:

* Features
* Content
* Interface
* Mods
* Balance
* Bugfixes
* Performance
* Infrastructure
* Build
* I18N

Example: ```SUMMARY: Content "Adds new mutation category 'Mouse'"```

See the [Changelog Guidelines](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md) for explanations of the categories.
-->
```SUMMARY: Balance "Update the volume and weight of planks to be more in line with real dimensional lumber"```
#### Purpose of change
<!--
If there's an existing issue describing the problem this PR addresses or the feature it adds, please link it like: ```#1234```
If it *fully* resolves an issue, link it like: Fixes #1234
Even if the issue describes the problem, please provide a few-sentence summary here.
Example: Fixes #1234 - XL mutants cannot wear arm/leg splints due to missing OVERSIZE flag.
If there is no related issue, please describe the issue you are addressing, including how to trigger a bug if this is a bugfix.
Don't put the backticks around the `#` and issue or pull request number to allow the GitHub automatically reference to it.
-->
Planks as they currently exist are either too heavy or too small to represent real, common dimensional lumber, with a density of 0.93g/mL whereas common lumber sorts vary (roughly) between 0.4-0.7g/mL, according to http://woodmonsters.com/density-chart.html or https://cedarstripkayak.wordpress.com/lumber-selection/162-2/ 

(Disclaimer: I didn't actually know the densities involved before starting this, but I worked out a starting point from Home Depot's given lumber dimensions and weights found here: https://www.homedepot.com/p/2-in-x-4-in-x-96-in-Premium-Kiln-Dried-Whitewood-Stud-161640/202091220)

#### Describe the solution
<!--
How does the feature work, or how does this fix a bug?
The easier you make your solution to understand, the faster it can get merged.
-->

I increased the volume of a plank from 1500mL to 4400mL and increased the weight of a plank from 1391g to 1400g. The latter is just for aesthetic purposes; I like round numbers and nine grams seems unlikely to go amiss, but it cuts down on the sig figs displayed, which I also like because we shouldn't be too precise about what exactly we're calling a plank.

Using the numbers from the Home Depot link above, if arranged like a 2x4 this new 'plank' is roughly 2"x4"x32", which fits the description decently and justifies it as a simple bashing weapon, whereas before it would have been right around a foot long, which is in no way a plank. Other dimensional arrangements are of course possible, but now the density and volume make sense of something you'd call a plank of construction wood instead a block of mahogany.


#### Describe alternatives you've considered
<!--
A clear and concise description of any alternative solutions or features you've considered.
-->

I considered changing the weight instead of the volume to rectify the density problem, or both the weight and volume to something closer to a real-life full 'plank' like you'd find at a hardware store, but chose against both options because this way we avoid introducing new mass conservation issues with recipe products.

#### Testing
<!--
Describe what steps you took to test that this PR resolved the bug or added the feature, and what tests you performed to make sure it didn't cause any regressions.
Also include testing suggestions for reviewers and maintainers.
-->

As this is just changing some numbers in wood.json, I didn't do any testing.

agm-114 commented:
I've went over the math myself, and these number add up, the board in question would be about 3ft long which nicely matches the length of a long string, and as long string's have their length specified it might be worthwhile to mention it.

I-am-Erk commented:
Although I like having the volume and size of a plank more precisely calculated, I don't recommend specifying what dimensions it is, as our recipes abstract planks to represent a wide variety of different dimensions of lumber. If we started specifying that they were 3' long we'd no longer be able to use them to frame walls, for example, and would need a new long plank item. That's getting a little more granular than we want.

I-am-Erk commented on data/json/items/resources/wood.json:
```diff
@@ -91,13 +91,13 @@
     "description": "A narrow, thick plank of wood, like a 2 by 4 or similar piece of dimensional lumber.  Makes a decent melee weapon, and can be used for all kinds construction.",
     "category": "spare_parts",
-    "weight": "1391 g",
+    "weight": "1400 g",
```
There's no need to revert this but in the future, generally I don't think there's a ton of value to changing things by such tiny amounts. It's not as though this change really added a lot of accuracy, a 2x4x32 inch piece of lumber could still weigh the original weight.

Jovus replied:
I definitely agree there was no need to change 1391 to 1400 except my personal liking for round numbers when they're available.

DaviBones commented:
Your math doesn't check out. Did you take into account that the actual dimensions of a 2x4 are  1.5" x 3.5"? (Don't worry, I don't blame you if not, it's a pretty stupid convention, but most lumber widths and heights are about 0.5 inches smaller than stated)

I-am-Erk commented:
The dimensions themselves are really irrelevant as stated above, but that density does not indeed check out. I'd set it to about 0.5 g/mL and keep the volume at 4400, so put it to about 2200g. A 3' long piece of 2x4 is pretty tiny, with the error DaviBones noticed that puts it at least to around 50" long by my reckoning (1.5"x3.5"x50" = 0.157 cubit ft = 4.445 liters), which is really a more reasonable length anyway

Jovus commented:
My one concern with increasing the weight to 2200g is that gross changes in the weight might cause weird issues with recipes further down the line. However, you all know more about the recipes than I do, and increasing the weight is better than decreasing it - weight can always be lost through manufacturing.

This leaves me with a question about this repo, though. Would it be better, for such a tiny change, to squash commit, or keep the history intact?

Jovus commented:
I've just changed plank weight to 2200g and volume to 4400ml and tried to squash commits to help keep the history clean for this PR. I also changed the name of the PR to be accurate, since I'd previously confused the weight and volume in the title of the PR (1400ml and 4400g instead of now 2200g and 4400ml).

Just FYI but I'm leaving for Switzerland tomorrow so if this doesn't get merged soon and needs my input I might be a bit more lengthy in response time, but I should still be able to shepherd this there. (I hope. Unless google throws a fit because I'll be logging in from a new computer.)

DaviBones commented:
> gross changes in the weight might cause weird issues with recipes further down the line

This is a good observation, and it's good to be thinking about these things. I did a workspace search for `"2x4"` and while there were way too many entries to look through them exhaustively, most usage of planks for non-construction crafting don't leave the plank intact -- e.g. wooden arrows/bolts, tool handles (whittled to a rod-shape), bows, etc. Those can be thought of as using a set amount of wood, thus don't care about the weight of the input plank.

The only exceptions that jump out at me are perhaps the two wooden "armor" pieces: `2byarm_guard` and `2byshin_guard`. You could certainly do a reality check on those as part of this PR, or even a separate PR, if you'd like, but I wouldn't say it's necessary or very important.

I-am-Erk commented:
The 2-by guards probably just need saws in their crafting requirements and splinters in their byproducts

Jovus commented:
I think that's a great idea for a follow-up PR, and I'll probably get on that after Christmas if I don't get swamped by work.
